app/src/main/java/org/openobservatory/ooniprobe/test/suite/ExperimentalSuite.java

```diff
@@ -39,15 +40,14 @@ public static ExperimentalSuite initForAutoRun() {
     public AbstractTest[] getTestList(@Nullable PreferenceManager pm) {
         if (super.getTestList(pm) == null) {
             ArrayList<AbstractTest> list = new ArrayList<>();
-            if (pm == null || pm.isExperimentalOn()){
+            // if (pm == null || pm.isExperimentalOn()){
```

Comment:
Please, don't commit commented out code unless there's a reason. Here it seems safe to delete these.

Reply:
All suite classes scheduled for deletion in https://github.com/ooni/probe-android/pull/637